src/components/FruitButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+function FruitButton({buttonType, buttonAction, buttonState, children}) {
+    return (
+        <button
+            type={buttonType}
+            onClick={buttonAction}
+            disabled={buttonState}
+        >
+            {children}
+        </button>
+    );
+}
```

Comment:
Dit zie ik nu als een doorgeefluik. Er wordt hier niets anders gedaan dan een ander component teruggeven en de props doorgegeven. Je had ook direct <button ... /> kunnen gebruiken en dan had je geen los component hoeven te maken. Mocht je bijv. een statisch classname setten, dan is het weer wél handig om een apart component te hebben, omdat je dat anders moet herhalen 😉

Reply:
@Zwen1111 , dus eigenlijk heb ik dit component helemaal niet nodig en had het op kunnen nemen in het fruitCounter component. Dit fruitCounter component had gewoon meer moeten omvatten (alles binnen het article zeg maar). Klopt dat ?